File: Problem Set 1/Mario-Less/mario.c
#include<cs50.h>
#include<stdio.h>

void print_row (int spaces, int bricks);

int main(void)
{   //prompt the user for the pyramid's height
    int n;
    do
    {
        n= get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //print a pyramid of that height
    for (int i = 0; i < n; i++)
    {
        //print row bricks
        print_row(n - i - 1, i+1);
        printf("\n");
    }

}

void print_row(int spaces, int bricks)
{   //print spaces
    for (int i = 0; i < spaces; i++)
    {
        printf(" ");
    }
    //print bricks
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
}

File: Problem Set 6/Sentimental-Mario-Less/mario.py
def print_row(spaces, bricks):
    # print spaces
    for _ in range(spaces):
        print(" ", end="")
    # print bricks
    for _ in range(bricks):
        print("#", end="")
    print()


def main():
    # prompt the user for the pyramid's height
    while True:
        try:
            n = int(input("Height: "))
            if n >= 1 and n <= 8:
                break
            else:
                print("Height must be between 1 and 8")
        except ValueError:
            print("Please enter a valid number: ")

    # print a pyramid of that height
    for i in range(n):
        # print rox bricks
        print_row(n - i - 1, i + 1)


if__name__ = "__main__"
main()
